examples/quick-start-sumologic-nodejs/readme.md

```diff
@@ -8,8 +8,12 @@ This is a simple quick start on how to configure a Node.js app to use OpenTeleme
 
 ### 1. Start the Tracetest Agent locally
 
+- Sign up on [`app.tracetest.io`](https://app.tracetest.io)
+- Create a new environment
+- Copy the start command under `Settings > Agent`
+
 ```bash
-tracetest start
+tracetest start --api-key ttagent_1234567890
```

Comment:
Philosophical question: should we add a placeholder here like `YOUR_API_KEY` instead of adding a fake `api-key`?

Having an almost "real" command like `tracetest start --api-key ttagent_1234567890` may lead users to copy and paste the command blindly, receive an error and don't understand what happened.

Reply:
We can do that. Makes sense.